Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -52,9 +52,11 @@ integrable, function space, l1
 
 noncomputable section
 
-open scoped Classical Topology BigOperators ENNReal MeasureTheory NNReal
+set_option linter.uppercaseLean3 false
```

Comment:
Can we add an exception to the global list instead of turning off the linter for the whole file?